base/utf16.jl

```diff
@@ -74,7 +64,7 @@ function encode16(s::AbstractString)
             push!(buf, UInt16(0xd7c0 + (c>>10)))
             push!(buf, UInt16(0xdc00 + (c & 0x3ff)))
         else
-            throw(ArgumentError("invalid Unicode character (0x$(hex(c)) > 0x10ffff)"))
+            throw(UnicodeError(UTF_ERR_INVALID_CHAR, 0, ch))
```

Comment:
This one's actually a tiny bit less information because of losing the ` > 0x10ffff`, right?